examples/reflection/generic_reflection.rs

```diff
@@ -13,6 +13,8 @@ fn main() {
 }
 
 #[derive(Reflect)]
+// Because `#[derive(Reflect)]` also derive `TypeName` for `MyType`,
+// the generic parameter must also implement `TypeName`.
```

Comment:
Macros should go below comments (which should probably also be doc comments for consistency with the other reflection examples).
```suggestion
/// Because `#[derive(Reflect)]` also derives `TypeName` for `MyType`,
/// the generic parameter must also implement `TypeName`.
#[derive(Reflect)]
```